src/renderer/components/archive/copyable.js

```diff
@@ -14,6 +14,7 @@ const Password = styled(ColoredDigits)`
   font-family: Anonymous;
   font-size: 14px;
   font-weight: bold;
+  user-select: all !important;
```

Comment:
I don't think `!important` is required as styled-components sets it with an exact class, as it does with all other components. This value should override any inherited style.

Reply:
Hmmm, I looked into why I had to do that the other day, and it looks like the `role="content"` attribute selector  in `global-styles.js` ends up being more/as specific as the styled-components class selector.

I guess I could do one of the following:
 - just forget about the selection behavior and not add the `user-select` style on the styled-component class selector
- Leave the `!important`
- Remove the `role` attribute on the password component

Reply:
I think the `!important` is fine in this case, imo.